static/src/javascripts/projects/common/modules/commercial/contributions-service.js

```diff
@@ -287,6 +299,27 @@ const renderEpic = async (module, meta) => {
     );
 };
 
+export const fetchPuzzlesData = async () => {
+    const page = config.get('page');
+    const payload = await buildBannerPayload();
+    const forcePuzzlesBannerTest = isInVariantSynchronous(puzzlesBanner, 'variant');
+    const isPuzzlesPage = page.section === 'crosswords' || page.series === 'Sudoku';
+
+    if (payload.targeting.shouldHideReaderRevenue || payload.targeting.isPaidContent) {
+        return Promise.resolve(null);
```

Comment:
If you’ve marked your function as `async`, you can return `null` directly.

Reply:
Thanks @mxdvl 👍